### ISSUE TITLE
No scheduler.step() call

### ISSUE THREAD

**Issue by P0lyFish** [yumingj/C2-Matching#5]
Hi, thanks for your interesting work.
I cannot find any scheduler.step() call in your code, even though there are several scheduler setups (In training logs, lr is always constant during the training process). Is this a bug or is it intentional?

**Comment by yumingj** [comment]
Hi, thanks for your interest in our work!

The `scheduler.step()` is called in the `update_learning_rate` funciton of `base_model.py`, line 165.

You can set the learning rate update scheme in config files. If you use my configuration, taking the training of stage 3 as an example, the learning rate would be changed after 72, 545 iterations and 211, 040 iterations.

**Comment by P0lyFish** [comment]
That's answer my question. I will close this issue.
